src/core/main.c

```diff
@@ -685,6 +685,33 @@ static int parse_config_file(void) {
         return 0;
 }
 
+static int manager_set_defaults(Manager *m) {
+        int r;
+
+        assert(m);
+
+        r = parse_config_file();
```

Comment:
Now `parse_config_file()` is invoked twice. I'd suggest to remove it here, make the function `void` and …